### PR TITLE
slight cleanup

### PR DESCRIPTION
**Pull request by vivianexists**
verryyyyy slight

### CODE REVIEW

**Comment by Glitchtest51** [comment]
passes all checks though i would change some things for the sake of readability lol

**Comment by vivianexists** [comment]
image

File: Python/scr/main.py [hunk 1]
```diff
@@ -1,6 +1,2 @@
 from Scripts.Emulator import Emu
-
-file = input('File Path:')
-
-emu = Emu(file)
-emu.main_loop()
\ No newline at end of file
+Emu(input('File Path:')).main_loop()
```
Comment (Glitchtest51) [inline Python/scr/main.py]:
```suggestion

ROM_Path = input('File Path:')
Emu(ROM_Path).main_loop()
```

File: Python/scr/Scripts/Display.py [hunk 2]
```diff
@@ -1,38 +1,27 @@
...
-        size = self.size
-        OFF = (0, 0, 0)
-        ON = (255, 255, 255)
-        self.screen.fill(OFF)
-        for y in range(self.height):
-            for x in range(self.width):
-                index = y * self.width + x
-                if self.screenArray[index] == 1:
-                    pygame.draw.rect(self.screen, ON, (x * size, y * size, size, size))
+        self.screen.fill((0, 0, 0))
```
Comment (Glitchtest51) [inline Python/scr/Scripts/Display.py]:
```suggestion
        OFF = (0, 0, 0)
        ON = (255, 255, 255)
        self.screen.fill(OFF)
```

File: Python/scr/Scripts/Display.py [hunk 3]
```diff
@@ -1,38 +1,27 @@
...
+        for i, pixel in enumerate(self.screenArray):
+            if pixel:
+                x, y = i % self.width, i // self.width
+                pygame.draw.rect(self.screen, (255, 255, 255),
```
Comment (Glitchtest51) [inline Python/scr/Scripts/Display.py]:
```suggestion
                pygame.draw.rect(self.screen, ON, (x * self.size, y * self.size, self.size, self.size))
```

File: Python/scr/Scripts/Display.py [hunk 4]
```diff
@@ -1,38 +1,27 @@
...
-            for x in range(self.width):
-                index = y * self.width + x
-                if self.screenArray[index] == 1:
-                    pygame.draw.rect(self.screen, ON, (x * size, y * size, size, size))
+        self.screen.fill((0, 0, 0))
+        for i, pixel in enumerate(self.screenArray):
+            if pixel:
+                x, y = i % self.width, i // self.width
+                pygame.draw.rect(self.screen, (255, 255, 255), 
+                               (x * self.size, y * self.size, self.size, self.size))
```
Comment (Glitchtest51) [inline Python/scr/Scripts/Display.py]:
```suggestion
```

**Review by Glitchtest51 — CHANGES_REQUESTED** [review]
readability since i use this file to remake it lol

**Comment by vivianexists** [comment]
yippee

**Review by Glitchtest51 — APPROVED** [review]
yippee

**Pull request merged**